services/token_update.proto

```diff
@@ -146,7 +146,7 @@ message TokenUpdateTransactionBody {
     /**
      * Metadata of the created token definition
      */
-    google.protobuf.BytesValue metadata = 16;
+    bytes metadata = 16;
```

Comment:
Should revert since you don't have to update this field in the request and want to differentiate between clear vs set.

Reply:
Thanks Steven. This is just a temporary workaround to unblock me on this branch until this pbj issue gets resolved -> https://github.com/hashgraph/pbj/issues/174